docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
 
 # Basic Components
 
-Nodes work with a data records. They can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
+Nodes work with data records. They can produce, fetch, send and collect data or organize the flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
 
-Most of the nodes, with source and sink nodes being notable exceptions, have both input and at least one output flow.
+Most of the nodes have both input and at least one output flow. Source and sink nodes are an exception.
 
 Sinks and filters can be disabled by selecting `Disable` checkbox.
 
 &nbsp;
 ## Variable
 
-A Variable component is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable holds its content and  its value is assigned to a newly declared `myFirstVariable` variable.
+A Variable component is used to declare a new variable. In its simplest form a variable declaration looks like the example  below. As the event has been read from a Kafka topic, the `#input` variable holds event contents. Event value is assigned to a newly declared `myFirstVariable` variable.
```

Comment:
```suggestion
A Variable component is used to declare a new variable. In its simplest form a variable declaration looks like the example  below. As the event has been read from a Kafka topic, the `#input` variable holds event contents. Here the event's value is assigned to a newly declared `myFirstVariable` variable.

Reply:
I agree with the suggestion to change, but I think "After that, the event's..." would be better, as Here sounds where ambiguous